scripts/sysman/events.yml

```diff
@@ -44,6 +44,10 @@ etors:
       desc: "Event is triggered when accelerator RAS uncorrectable errors cross thresholds (use $sRasSetConfig() to configure - disabled by default)."
     - name: DEVICE_RESET_REQUIRED
       desc: "Event is triggered when the device needs to be reset (use $sDeviceGetState() to determine the reasons for the reset)."
+    - name: SURVIVABILITY_MODE_DETECTED
+      desc: 
+            "1.9": "Event is triggered when graphics driver encounter an error condition."
```

Comment:
format needs to be:
```
name: SURVIVABILITY_MODE_DETECTED
desc: "Event is triggered when graphics driver encounter an error condition."
version: "1.9"
```

For example: check temperature.yml file for "GPU_BOARD" enum